src/tests/ut_lind_fs_broken_close.rs

```diff
@@ -0,0 +1,10 @@
+
+#[cfg(tests)]
```

Comment:
not sure whats going on here

Reply:
This was where I was going to start writing tests. It does not have to be implemented, though.